### DISCUSSION TITLE
Question… symbols and key frames

### DISCUSSION THREAD

**Discussion by SharpsterDraws** [keiwando/toonsquid#691] [General]
Is there a way to split or cut a layer and not have the keyframes duplicate automatically? 

It sucks when I’m keyframing a mouth symbol on a layer and I cut or split the layer and all the keyframes just duplicate.

**Comment by budaloops** [comment]
Happens in other apps too. Might be a norm 🤷 One solution is to plan ahead, because in spite of hitting these roadblocks 🧱, once we keep playing around with the timeline and finding out what’s not working 🤌 it’s gonna become clearer what makes our gifts possible 🎁 

What are you working on?

**Reply by SharpsterDraws** [reply]
I was just hoping there was a clear all keyframes button or similar solution…I’m working on my Dracula Vs Wolfman animation… should have it finished today.

**Reply by SharpsterDraws** [reply]
Here’s an example of my animations https://youtu.be/Xp-6gO74oQk?si=i_knryksk0dRWkmB

**Reply by budaloops** [reply]
Sweet! We can look forward to the finished piece on your channel
